datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validators/validator.py

```diff
@@ -183,6 +183,12 @@ def extract_oid_from_symbol(self, metric, file):
         Function to extract OID from symbol
         """
         OID = metric.get('symbol').get('OID')
+        if (
+            metric.get('forced_type') == 'flag_stream'
+            and metric.get('options')
+            and metric.get('options').get('metric_suffix')
+        ):
+            OID = OID + metric.get('options').get('metric_suffix')
```

Comment:
So, the `OID` value will be something like `1.2.3suffix`? it should work, but might look a bit strange workaround.

Reply:
1.2.3suffix => 1.2.3.<PLACEMENT>

Reply:
Yes. Now it is as `OID.placement` so a exemple would be if OID is 1.2.3 and the placement is 9:
the value would be 1.2.3.9